plugins/woocommerce/src/Internal/Admin/ReviewsCommentsOverrides.php

```diff
@@ -21,9 +21,39 @@ class ReviewsCommentsOverrides {
 	 */
 	public function __construct() {
 
+		add_action( 'admin_notices', [ $this, 'display_notices' ] );
+
 		add_filter( 'comments_list_table_query_args', [ $this, 'exclude_reviews_from_comments' ] );
 	}
 
+	/**
+	 * Renders admin notices.
+	 */
+	public function display_notices() {
+		$screen = get_current_screen();
+
+		if ( empty( $screen ) || 'edit-comments' !== $screen->base ) {
+			return;
+		}
+
+		$this->display_reviews_moved_notice();
+	}
+
+	/**
+	 * Renders an admin notice informing the user that reviews were moved to a new page.
+	 */
+	protected function display_reviews_moved_notice() {
+		?>
+
+		<div class="notice notice-info is-dismissible">
```

Comment:
Shouldn't the dismissible state be persisted once the notice is dismissed?

https://cloud.skyver.ge/6quZbP4D